data/Sandeep277.json

```diff
@@ -0,0 +1,25 @@
+
+{
+  "name": "Sandeep Kushwaha",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Student",
+  "avatar": "https://github.com/Sandeep277.png",
```

Comment:
you can remove the avatar line,  it will directly fetch  your real time image from your github.

Reply:
now avatar line is removed 
thanks for your feedback